src/test/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSparkIntegrationTest.java

```diff
@@ -0,0 +1,114 @@
+            this.alignerRefIndexImgLoc = alignerRefIndexImgLoc;
+            this.alignmentOutput = alignmentOutput;
+        }
+
+        String getCommandLineNoApiKey() {
+            return " --inputAssemblyDir " + inputAssemblies +
+                    " -O " + alignmentOutput +
+                    " --bwamemIndexImage " + alignerRefIndexImgLoc;
+        }
+
+        String getCommandLine() {
+            return  getCommandLineNoApiKey() +
+                    " --apiKey " + getGCPTestApiKey();
+        }
+    }
+
+    @DataProvider(name = "alignAssembledContigsSparkIntegrationTest")
+    public Object[][] createTestData() throws IOException {
+        List<Object[]> tests = new ArrayList<>();
+
+        final File tempWorkingDir = BaseTest.createTempDir("whatever");
+        tempWorkingDir.deleteOnExit();
+
+        final File assemblyWithPackedFastaWithLength = Files.createDirectory(Paths.get(tempWorkingDir.getAbsolutePath()+"/"+"assemblyWithPackedFastaWithLength")).toFile();
+        try(  final PrintWriter out = new PrintWriter( new File(assemblyWithPackedFastaWithLength, "contents.txt") )  ){
+            out.println( ContigCollectionTest.contigsCollectionPackedFastaAndWithAsmId[0][1] );
+        }
+        tests.add(new Object[]{new AlignAssembledContigsSparkIntegrationTest.AlignAssembledContigsSparkIntegrationTestArgs(assemblyWithPackedFastaWithLength.getAbsolutePath(),
+                tempWorkingDir.getAbsolutePath()+"/"+"outputWithLength",
+                SVIntegrationTestDataProvider.ALIGNER_INDEX_IMG)});
+
+        final File assemblyWithPackedFastaWithoutLength = Files.createDirectory(Paths.get(tempWorkingDir.getAbsolutePath()+"/"+"assemblyWithPackedFastaWithoutLength")).toFile();
+        try(  final PrintWriter out = new PrintWriter( new File(assemblyWithPackedFastaWithoutLength, "contents.txt") )  ){
+            out.println( ContigCollectionTest.contigsCollectionPackedFastaAndWithAsmId[0][1] );
+        }
+        tests.add(new Object[]{new AlignAssembledContigsSparkIntegrationTest.AlignAssembledContigsSparkIntegrationTestArgs(assemblyWithPackedFastaWithoutLength.getAbsolutePath(),
+                tempWorkingDir.getAbsolutePath()+"/"+"outputWithoutLength",
+                SVIntegrationTestDataProvider.ALIGNER_INDEX_IMG)});
+
+        return tests.toArray(new Object[][]{});
+    }
+
+    @Test(dataProvider = "alignAssembledContigsSparkIntegrationTest", groups = "sv")
+    public void testAlignAssembledContigsSparkRunnableLocal(final AlignAssembledContigsSparkIntegrationTest.AlignAssembledContigsSparkIntegrationTestArgs params) throws IOException {
+        new IntegrationTestSpec(
+                new ArgumentsBuilder().add(params.getCommandLineNoApiKey()).getString(),
+                SVIntegrationTestDataProvider.dummyExpectedFileNames)
+                .executeTest("testAlignAssembledContigsSparkRunnableLocal-", this);
+    }
+
+    @Test(dataProvider = "alignAssembledContigsSparkIntegrationTest", groups = "sv")
```

Comment:
The `MiniCluster` is not the Spark cluster that I assumed it to be, it is essentially a mock up of HDFS, hence the grouping.